README.md

```diff
@@ -67,6 +67,67 @@ functions:
         comparisonOperator: GreaterThanOrEqualToThreshold
 ```
 
+## Multiple topic definitions
+
+You can define several topics for alarms. For example you want to have topics for critical alarms
+reaching your pagerduty, and different topics for noncritical alarms, which just send you emails.
+
+In each alarm definition you have to specify which topics you want to use. In following example
+you get an email for each function error, pagerduty gets alarm only if there are more than 20
+errors in 60s
+
+```yaml
+custom:
+  alerts:
+
+    topics:
+      critical:
+        ok:
+          topic: ${self:service}-${self:custom.stage}-critical-alerts-ok
+          notifications:
+          - protocol: https
+            endpoint: https://events.pagerduty.com/integration/.../enqueue
+        alarm:
+          topic: ${self:service}-${self:custom.stage}-critical-alerts-alarm
```

Comment:
should `${self.custom.stage}` be `${opt:stage}` ?

Reply:
Yeah - you're right. The rest of readme.md uses opt:stage, so it should be the same - fixed now.

We use a bit different naming convetion - I missed that..